openstudiocore/src/osversion/VersionTranslator.cpp

```diff
@@ -4757,6 +4759,96 @@ std::string VersionTranslator::update_2_8_1_to_2_9_0(const IdfFile& idf_2_8_1, c
 
 }
 
+
```

Comment:
And here I was all revved up to complain about the required version translation that I was sure you would have forgotten, but nope. you got that too. I'm gonna have to up my game to catch issues in your PRs!

Reply:
I'd actually like to create a PR template with a review checklist that includes stuff like this which is easy to forget